ambari-server/src/main/java/org/apache/ambari/server/update/HostUpdateHelper.java

```diff
@@ -566,15 +576,14 @@ public static void main(String[] args) throws Exception {
 
       LOG.info("Host names update completed successfully.");
 
-      hostUpdateHelper.stopPersistenceService();
-    } catch (Throwable e) {
-      if (e instanceof AmbariException) {
+    } catch (AmbariException e) {
         LOG.error("Exception occurred during host names update, failed", e);
-        throw (AmbariException)e;
-      }else{
+        throw e;
+    } catch (Throwable e){
```

Comment:
Any particular reason why we're catching Throwable and converting it to an exception? This will catch errors like OutOfMemoryError too.

Reply:
fixed, thanks